editor/src/components/editor/actions/actions.tsx

```diff
@@ -5608,7 +5471,7 @@ function saveFileInProjectContents(
   }
 }
 
-type ElementToInsert = {
+export type PasteElementToInsert = {
```

Comment:
It doesn't look like this (or the function below) need to be exported

Reply:
fixed